nginx.conf

```diff
@@ -5,7 +5,10 @@ events { worker_connections 1024; }
 
 http {
   charset utf-8;
-  log_format cloudfoundry '$http_x_forwarded_for - $http_referer - [$time_local] "$request" $status $body_bytes_sent';
```

Comment:
Are you deliberately eliminating the http_x_forwarded_for header? I've found that one useful for troubleshooting in past situations, but maybe between CF and CDN it isn't?

Reply:
It's already included in the router logs, as is some of the info we're still including in the app logs.

Reply:
That'll do then. Thanks!